tests/verifyAWSRDSMySQLInstance_test.js

```diff
@@ -80,3 +82,40 @@ Scenario(
     }
   },
 );
+
+Scenario(
+  'PMM-T716 - Verify adding PostgreSQL RDS monitoring to PMM via UI @not-pr-pipeline',
+  async ({
+    I, remoteInstancesPage, pmmInventoryPage, qanPage, qanFilters, qanOverview, dashboardPage,
+  }) => {
+    const serviceName = 'pmm-qa-postgres-12';
+
+    I.amOnPage(remoteInstancesPage.url);
+    remoteInstancesPage.waitUntilRemoteInstancesPageLoaded().openAddAWSRDSMySQLPage();
+    remoteInstancesPage.discoverRDS();
+    remoteInstancesPage.verifyInstanceIsDiscovered(serviceName);
+    remoteInstancesPage.startMonitoringOfInstance(serviceName);
+    remoteInstancesPage.verifyAddInstancePageOpened();
+    await remoteInstancesPage.checkField(remoteInstancesPage.fields.serviceName, serviceName);
```

Comment:
this is not what [step 5](https://jira.percona.com/secure/Tests.jspa#/testCase/PMM-T716) says to do.